synapse/types.py

```diff
@@ -751,3 +751,32 @@ def get_verify_key_from_cross_signing_key(key_info):
     # and return that one key
     for key_id, key_data in keys.items():
         return (key_id, decode_verify_key_bytes(key_id, decode_base64(key_data)))
+
+
+@attr.s(auto_attribs=True, frozen=True, slots=True)
+class UserInfo:
+    """Represents an object with information about a user.
```

Comment:
It *is* an object with information about a user.

```suggestion
    """Holds information about a user. Result of get_userinfo_by_id.
```